force-app/main/default/lwc/bulkServiceDeliveryUI/bulkServiceDeliveryUI.html

```diff
@@ -16,6 +16,9 @@
         </div>
 
         <lightning-card if:false={showWizard}>
+            <div class="slds-var-p-horizontal_medium">
+                {labels.serviceFieldMessage}
```

Comment:
@AndersonTarren, This message is currently not read out by the screen reader. We may want to add aria-live to the div for the screen reader to read the message out.